CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+### 1.15.0
```

Comment:
I think that these two commits don't have a corresponding entry in the README:
- https://github.com/redhat-developer/yaml-language-server/commit/02d69fe2908e68184ed130b936b4e0107122620e
- https://github.com/redhat-developer/yaml-language-server/commit/2f2683b105229dc79734dd84149450a5d43bb4e4

Reply:
@datho7561 https://github.com/redhat-developer/yaml-language-server/commit/02d69fe2908e68184ed130b936b4e0107122620e was closed without merge the [PR](https://github.com/redhat-developer/yaml-language-server/pull/923) and https://github.com/redhat-developer/yaml-language-server/commit/2f2683b105229dc79734dd84149450a5d43bb4e4 was closed without merge the [PR](https://github.com/redhat-developer/yaml-language-server/pull/924). So I haven't made an entry

Reply:
aww, sorry missed the logs. Done. Please have a look